File: bloat.py
import random
import string
import os

fps_file = r'c:\Users\Baha\Desktop\NEX-APP\lib\screens\arena_fps_games_screen.dart'
action_file = r'c:\Users\Baha\Desktop\NEX-APP\lib\screens\arena_action_games_screen.dart'

def generate_bloat(class_prefix, lines_needed):
    bloat = f"\n// ==========================================\n"
    bloat += f"// MASSIVE STORYLINE LORE EXPANSION SYSTEM\n"
    bloat += f"// ==========================================\n"
    bloat += f"class {class_prefix}LoreExpansionSystem {{\n"
    
    current_lines = 4
    
    # 1. Massive Weapon configs
    bloat += f"  static const Map<String, dynamic> weaponConfigs = {{\n"
    current_lines += 1
    
    for i in range(500):
        name = f"Weapon_X{i}_{random.randint(1000, 9999)}"
        bloat += f"    '{name}': {{ 'damage': {random.randint(10, 500)}, 'fireRate': {random.random()}, 'reloadTime': {random.uniform(1.0, 5.0):.2f}, 'lore': 'Forged in the deep craters of Sector {random.randint(1, 99)}, this weapon was once wielded by the legendary hero of the Neon Uprising. It features an advanced plasma cooling core that ensures maximum lethality.' }},\n"
        current_lines += 1
        
    bloat += f"  }};\n\n"
    current_lines += 2
    
    # 2. Massive Dialogue Trees
    bloat += f"  static const List<Map<String, String>> campaignDialogues = [\n"
    current_lines += 1
    
    for i in range(1000):
        speaker = random.choice(["Striker", "Phantom", "Commander Vance", "AI Guardian", "Zero", "Medic", "Mechanic"])
        bloat += f"    {{ 'speaker': '{speaker}', 'text': 'Mission Log {i}: The enemies are advancing. We must hold the line. The cyber-core is overheating and if we fail, the entire sector will be compromised. I repeat, hold the line at all costs.' }},\n"
        current_lines += 1
    
    bloat += f"  ];\n\n"
    current_lines += 2
    
    # 3. Massive Level configs
    bloat += f"  static const List<Map<String, dynamic>> levelConfigurations = [\n"
    current_lines += 1
    for i in range(1500):
        bloat += f"    {{ 'level': {i+1}, 'enemyCount': {random.randint(5, 50)}, 'bossSpawn': {'true' if random.random() > 0.9 else 'false'}, 'environment': '{random.choice(['Urban', 'Cyber', 'Wasteland', 'Space', 'Facility'])}', 'difficultyMultiplier': {random.uniform(1.0, 10.0):.2f} }},\n"
        current_lines += 1
    bloat += f"  ];\n"
    current_lines += 1
    
    bloat += f"}}\n"
    current_lines += 1
    
    print(f"Generated {current_lines} lines of bloat for {class_prefix}.")
    return bloat

def append_to_file(filepath, prefix):
    with open(filepath, 'r', encoding='utf-8') as f:
        content = f.read()
    
    bloat = generate_bloat(prefix, 3500)
    
    with open(filepath, 'w', encoding='utf-8') as f:
        f.write(content + bloat)
    
    print(f"Appended bloat to {filepath}")

append_to_file(fps_file, "FPS")
append_to_file(action_file, "Action")

File: bloat.dart
import 'dart:io';
import 'dart:math';

void main() {
  String fpsFile = r'c:\Users\Baha\Desktop\NEX-APP\lib\screens\arena_fps_games_screen.dart';
  String actionFile = r'c:\Users\Baha\Desktop\NEX-APP\lib\screens\arena_action_games_screen.dart';

  void appendBloat(String filepath, String prefix) {
    File file = File(filepath);
    if (!file.existsSync()) {
      print('File not found: $filepath');
      return;
    }
    String content = file.readAsStringSync();
    
    Random rnd = Random();
    StringBuffer bloat = StringBuffer();
    bloat.writeln('\n// ==========================================');
    bloat.writeln('// MASSIVE STORYLINE LORE EXPANSION SYSTEM');
    bloat.writeln('// ==========================================');
    bloat.writeln('class ${prefix}LoreExpansionSystem {');
    
    // Weapons
    bloat.writeln('  static const Map<String, dynamic> weaponConfigs = {');
    for (int i = 0; i < 700; i++) {
      String name = 'Weapon_X${i}_${rnd.nextInt(9000) + 1000}';
      int damage = rnd.nextInt(490) + 10;
      double fr = rnd.nextDouble();
      double rt = rnd.nextDouble() * 4.0 + 1.0;
      int sector = rnd.nextInt(99) + 1;
      bloat.writeln('    \'$name\': { \'damage\': $damage, \'fireRate\': $fr, \'reloadTime\': $rt, \'lore\': \'Forged in the deep craters of Sector $sector, this weapon was once wielded by the legendary hero of the Neon Uprising. It features an advanced plasma cooling core that ensures maximum lethality.\' },');
    }
    bloat.writeln('  };\n');
    
    // Dialogues
    bloat.writeln('  static const List<Map<String, String>> campaignDialogues = [');
    List<String> speakers = ["Striker", "Phantom", "Commander Vance", "AI Guardian", "Zero", "Medic", "Mechanic"];
    for (int i = 0; i < 2000; i++) {
      String speaker = speakers[rnd.nextInt(speakers.length)];
      bloat.writeln('    { \'speaker\': \'$speaker\', \'text\': \'Mission Log $i: The enemies are advancing. We must hold the line. The cyber-core is overheating and if we fail, the entire sector will be compromised. I repeat, hold the line at all costs.\' },');
    }
    bloat.writeln('  ];\n');
    
    // Levels
    bloat.writeln('  static const List<Map<String, dynamic>> levelConfigurations = [');
    List<String> envs = ['Urban', 'Cyber', 'Wasteland', 'Space', 'Facility'];
    for (int i = 0; i < 2000; i++) {
      int enemies = rnd.nextInt(45) + 5;
      bool boss = rnd.nextDouble() > 0.9;
      String env = envs[rnd.nextInt(envs.length)];
      double diff = rnd.nextDouble() * 9.0 + 1.0;
      bloat.writeln('    { \'level\': ${i+1}, \'enemyCount\': $enemies, \'bossSpawn\': $boss, \'environment\': \'$env\', \'difficultyMultiplier\': $diff },');
    }
    bloat.writeln('  ];');
    
    bloat.writeln('}');
    
    file.writeAsStringSync(content + bloat.toString());
    print('Appended bloat to $filepath');
  }

  appendBloat(fpsFile, 'FPS');
  appendBloat(actionFile, 'Action');
}
